pageserver/src/repository.rs

```diff
@@ -182,14 +182,12 @@ impl Value {
 
 #[derive(Clone, Copy, Debug)]
 pub enum TimelineSyncStatusUpdate {
-    Uploaded,
```

Comment:
Turned out we did not react on this status change at all, so I've removed it, but left the enum to cause a bit less ruckus in this PR and to anticipate future changes that might require adding more variants here.